<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>pix1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>pix2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>80</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>pix3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>90</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>pix4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_Sun">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>-80</y>
      <width>251</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>pixSun</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>1081</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
